### ISSUE TITLE
Styling support on win32 platforms using ecma ansi

### ISSUE THREAD

**Issue by jquast** [jquast/blessed#18]
## Introduction

Currently PDCurses builds of python **does work** with blessed, thanks to help from @M-griffin, but there is _no termcap_ database.

This means an empty string (`""`) is always returned for bold, colors, etc. This is directly related to  erikrose/blessings#21
## Acceptance Criteria

All basic capabilities supported by https://github.com/tehmaze/ansi are supported on Windows. That is, `Terminal().bold_red("Hello")` should emit the correct ECMA-48 ANSI escape sequences on the windows platform. Although not included in scope, it is assumed that by exporting ANSI escape sequences, that it should be possible to combine  with https://pypi.python.org/pypi/colorama
## Furthermore

The root of the problem is that PDCurses, a BSD licensed curses implementation, does not provide a terminal database parser, nor a terminal database records file, on the windows platform.  The true solution is to simply implement it, but this is actually quite difficult as reported by @tehmaze who has investigated writing a from-scratch termcap parser.

In the future, it is viable that the blessed project does not import curses at all -- and parses termcap in pure python. There isn't any reason why it can't, it simply re-uses and links with C Curses libraries to prevent code duplication.  However, a pure-python implementation of tigetstr(3x) and friends would allow blessed to be portable, independent of the availability of the curses module !

**Comment by jquast** [comment]
The following code,

https://github.com/PowerShell/Win32-OpenSSH/commit/7aac59e524e6dc9f1c1faed18340cff7dfd9a47c#diff-1649f727622a11d5351a0131786f82b2R422

Answers the question, "what terminal sequences will windows clients support when using terminal type ansi[1]"?

1: I assume the client code will announce TERM=ansi, it is yet to be seen, they are probably exporting such environment value manually, but ansi and vt52 is all the given code can support:

```
+					if (term_mode == TERM_ANSI)
+					else if (term_mode == TERM_VT52)
```

**Comment by jquast** [comment]
http://www.nivot.org/blog/post/2016/02/04/Windows-10-TH2-%28v1511%29-Console-Host-Enhancements

**Comment by jquast** [comment]
Also, we've approached a future where the terminfo database is... not needed. We have only a few simple terminal types, ansi, xterm, xterm-256color. It is very rare to see other terminal types, and most are simply derived anyway, such as ansi-bbs or rxvt.

**Comment by jquast** [comment]
Closed by #110
